main.tf

```diff
@@ -1,6 +1,6 @@
 locals {
   module_name    = "transformer-kinesis-ec2"
-  module_version = "0.2.1"
+  module_version = "0.2.2"
 
   app_name    = "transformer-kinesis"
   app_version = "5.2.0"
```

Comment:
If we bump to 5.2.1 then we get event table created on startup (https://github.com/snowplow/snowplow-rdb-loader/issues/1034), so we make Snowflake/Databricks loader setups simpler.  It came out around/after this PR.

This could be a separate issue / release so we can get this one released.

Reply:
Tested with proposed change while testing out Databricks change - works fine.